cmd/up/up.go

```diff
@@ -361,18 +364,13 @@ func Up() *cobra.Command {
 }
 
 // AddArgs sets the args as options and return err if it's not compatible
-func (o *UpOptions) AddArgs(cmd *cobra.Command, args []string) error {
+func (o *UpOptions) AddArgs(args []string) error {
 
-	maxV1Args := 1
-	docsURL := "https://okteto.com/docs/reference/cli/#up"
-	if len(args) > maxV1Args {
-		cmd.Help()
-		return oktetoErrors.UserError{
-			E:    fmt.Errorf("%q accepts at most %d arg(s), but received %d", cmd.CommandPath(), maxV1Args, len(args)),
-			Hint: fmt.Sprintf("Visit %s for more information.", docsURL),
-		}
-	} else if len(args) == 1 {
+	if len(args) == 1 {
```

Comment:
We need to distinct if the first arg is a command or a service. 

Maybe we need to move this function after we have loaded the manifest and check the first argument vs the services defined under the dev section

Reply:
So are we taking into consideration that the command passed in `okteto up <command>` would be applicable to all dev containers in comparison to `okteto up [svc] <command>` where the command will be run in the particular dev container?

Reply:
okteto up can only be run on one service.

We need to distinct the following use cases:
- `okteto up`: We will show the selector if more than one dev is in the manifest and run the command defined there
- `okteto up arg1`: There are two options here, the first one the service is defined in the manifest and the second one where the svc is not defined
  -  If it's defined we need to use that dev environment and don't show the prompt. The command would be the one defined in the manifest. For example (`okteto up api`)
  - If it's not defined we'll show the selector if more than one dev is available and use the arg as the command for our dev environment. For example (`okteto up sh`)
- `okteto up arg1 arg2`. As the previous one we need to check if the first arg is a dev environment or not.
  -  If it's defined we need to use that dev environment and don't show the prompt. The second arg will be used as the command of the dev environment. For example (`okteto up api sh`)
  - If it's not defined we'll show the selector if more than one dev is available and use the args as the command for our dev environment. For example (`okteto up sh echo hola`)

Reply:
So, we'll always check to see if the first arg is a service or not. 

* If yes, we use that dev environment and then run commands (if extra args exist)
* If not, we show the prompt and pass all the args as commands to be run

If there aren't any args at all then keep doing what was done before.

Reply:
@abs007 that sounds good

Reply:
@pchico83 @jLopezbarb wouldn't it be better to add the command as an option like `--command bash`?

I've seen that @abs007 has done a great work handling the distinction between a service and a command, but I feel like it's very error prone since we do a lot of modifications to the `manifest` struct

Reply:
> @pchico83 @jLopezbarb wouldn't it be better to add the command as an option like `--command bash`?

We need the command to be an array of strings. If we add it as a flag the user will have to add several `--command`. That's why all the other CLIs like `kubectl` or `docker` use it as the final argument like `docker exec -ti my_container -- "echo a && echo b"`